source/code-generation/query-converter-test-bench.txt

```diff
@@ -10,10 +10,43 @@ Query Converter Test Bench
    :depth: 1
    :class: singlecol
 
-Stub page.
+Use Query Converter Test Bench to test SQL queries 
+and stored procedures converted by :ref:`rm-query-converter`. 
+Query Converter Test Bench uses the connections defined for your 
+source and destination databases and allows you to run 
+the original and converted queries or stored procedures.
+The results from each display on the Query Converter 
+Test Bench allowing for a side-by-side comparison.
+
+Use Cases
+---------
+
+Query converted test bench allows you to:
+
+- Compare your original and converted 
+  query results in a single place.
+- Increase your confidence in code automatically 
+  converted by the query converter.
+- See potential logic errors and edit the converted 
+  code before moving it to production.
+
+Requirements
+------------
+
+To use Query Converter Test Bench, you must 
+run the Query Runner Docker container. This 
+container can be run locally or hosted remotely. 
+For details, see :ref:`rm-install-query-runner`.
+
+Get Started
+-----------
+
+- :ref:`rm-install-query-runner` 
+- :ref:`rm-compare-converted-queries`
 
```

Comment:
[Suggestion]
Could we add the Learn More section and link to the Query Converter page here again?

Reply:
Makes senses, thanks!